Xamarin.Forms.Core/VisualElement.cs

```diff
@@ -54,6 +54,64 @@ public partial class VisualElement : Element, IAnimatable, IVisualElementControl
 
 		public static readonly BindableProperty ScaleYProperty = BindableProperty.Create(nameof(ScaleY), typeof(double), typeof(VisualElement), 1d);
 
+		internal static readonly BindableProperty TransformProperty = BindableProperty.Create("Transform", typeof(string), typeof(VisualElement), null, propertyChanged: OnTransformChanged);
+
+		static void OnTransformChanged(BindableObject bindable, object oldValue, object newValue)
+		{
+			if ((string)newValue == "none") {
+				bindable.ClearValue(TranslationXProperty);
+				bindable.ClearValue(TranslationYProperty);
+				bindable.ClearValue(RotationProperty);
+				bindable.ClearValue(RotationXProperty);
+				bindable.ClearValue(RotationYProperty);
+				bindable.ClearValue(ScaleProperty);
+				bindable.ClearValue(ScaleXProperty);
+				bindable.ClearValue(ScaleYProperty);
+				return;
+			}
+				var transforms = ((string)newValue).Split(' ');
+			foreach (var transform in transforms) {
+				if (string.IsNullOrEmpty(transform) || transform.IndexOf('(') < 0 || transform.IndexOf(')') < 0)
+					throw new FormatException("Format for transform is 'none | transform(value) [transform(value) ]*'");
+				var transformName = transform.Substring(0, transform.IndexOf('('));
+				var value = transform.Substring(transform.IndexOf('(') + 1, transform.IndexOf(')') - transform.IndexOf('(') - 1);
+				if (transformName.StartsWith("translateX", StringComparison.InvariantCulture) && double.TryParse(value, out var translationX))
+					bindable.SetValue(TranslationXProperty, translationX);
+				else if (transformName.StartsWith("translateY", StringComparison.InvariantCulture) && double.TryParse(value, out var translationY))
+					bindable.SetValue(TranslationYProperty, translationY);
+				else if (transformName.StartsWith("translate", StringComparison.InvariantCulture)) {
+					var translate = value.Split(',');
+					if (double.TryParse(translate[0], out translationX) && double.TryParse(translate[1], out translationY)) {
```

Comment:
Need to `out var` these to prevent
```

Xamarin.Forms.Core\VisualElement.cs(79,46): Error CS0165: Use of unassigned local variable 'translationX'

Xamarin.Forms.Core\VisualElement.cs(81,46): Error CS0165: Use of unassigned local variable 'translationY'
```

Reply:
nope, wrong line. we can't `out var` here as the variable is already defined in the scope

the error are on line 79 and 81